### ISSUE TITLE
Get markets ids

### ISSUE THREAD

**Issue by pedroloureiro82** [danieledangeli/betfair-php#14]
Hi, how do i get the markets ids from an event? And after i have the market id, how can i get the odds of that market?

**Comment by danieledangeli** [comment]
Hi @pedrogrou. It's possible to implement this process with the library. Have a look here: https://github.com/danieledangeli/betfair-php/blob/master/examples/SearchMarketBookExample.php where the odds are retrieved given a market Ids object. Each event has more "marketIds". To do that just read this documentation here: https://api.developer.betfair.com/services/webapps/docs/display/1smk3cen4v3lu3yomq5qye0ni/listMarketCatalogue and use the proper object in the library: "MarketCatalogue" where you can specify an eventId and get all the marketIds of that event. I have an helper method there: "getMarketCatalogueFilteredByEventIds(array eventIds)" which return the marketCatalogue of the events.
In a nutshell:
Let's consider your eventId = "1.8913"

```
$marketCatalogueHelper = $betfair->getMarketCatalogue();
$marketCatalogue = $marketCatalogueHelper->getMarketCatalogueFilteredByEventIds(["1.8913"])
```

**Comment by pedroloureiro82** [comment]
Thank you @danieledangeli.

It worked. Just a small correction in code:

```
$marketCatalogueHelper = $betfair->getBetfairMarketCatalogue();
$marketCatalogue = $marketCatalogueHelper->getMarketCatalogueFilteredByEventIds(["1.8913"]);
```

Thank you very much for your work.

**Comment by danieledangeli** [comment]
:+1:
